### PR TITLE
Add homepage -- LIBWEB-395

### PR DESCRIPTION
**Pull request by jonearley**

### CODE REVIEW

File: plugins/gatsby-source-umich-lib/gatsby-node.js [hunk 1]
```diff
@@ -49,7 +49,19 @@ exports.sourceNodes = async (
...
+    type HTML {
+      processed: String
+    }
+
+    type paragraph__hero_panel implements Node {
+      field_caption_text: HTML
+    }
+  `
+
+  createTypes(typeDefs)
```
Comment (jonearley) [inline plugins/gatsby-source-umich-lib/gatsby-node.js]: @vagabundeo I don't think it's possible to make fields with depth generically nullable. So added a definition here to make sure `field_caption_text` exists on `paragraph__hero_panel`.

Comment (vagabundeo) [reply plugins/gatsby-source-umich-lib/gatsby-node.js]: eh. this will bite us if we have to do it every time

Comment (jonearley) [reply plugins/gatsby-source-umich-lib/gatsby-node.js]: I get what you feel you want, but it's not feasible.

Comment (jonearley) [reply plugins/gatsby-source-umich-lib/gatsby-node.js]: I don't see how Gatsby could assume the shape of type definitions is has no information about.

**Pull request merged**